docs/orchestrating-elastic-stack-applications/security/auth-configs-using-stack-config-policy.asciidoc

```diff
@@ -0,0 +1,20 @@
+:page_id: auth-config-using-stack-config-policy
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Authentication policies using Elastic Stack configuration policy
+
+NOTE: This requires a valid Enterprise license or Enterprise trial license. Check <<{p}-licensing,the license documentation>> for more details about managing licenses.
+
+Starting from ECK `2.11.0` new options were added that enable adding configurations that go into `elasticsearch.yml` and `kibana.yml` to the Elastic Stack configuration policy.
```

Comment:
I would probably phrase it a little bit more with focus on what users can do with it. 

```suggestion
ECK `2.11.0` extends the functionality of [link...]Elastic Stack configuration policies so that it becomes possible to configure Elasticsearch security realms for more than one Elastic stack at once. 
```

Reply:
Made the suggested change.